src/legacy/ui/public/vis/editors/default/_agg_params.scss

```
@@ -10,3 +10,12 @@
   color: $textColor;
   background-color: $backgroundColor;
 }
+
+.visEditorAggParam--inline {
+  display: inline-block;
+  width: 50%;
+}
+
+.visEditorAggParam--inline-size {
```

As stated above about the class names this can be changed to:

```suggestion
.visEditorAggParam--half + .visEditorAggParam--half {
```

---

😢